File: CodigoEmJS/q2.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

function contarA(s) {
    const sLower = s.toLowerCase();
    const contagem = (sLower.match(/a/g) || []).length; // Usando regex para contar 'a'
    return contagem;
}

// Usando readline para entrada do usuário
rl.question("Digite uma string: ", (string) => {
    const contagem = contarA(string);
    console.log(`A letra 'a' ocorre ${contagem} vez(es) na string.`);
    rl.close(); // Fecha a interface após a entrada
});

// PARA RODAR O CODIGO NO TERMINAL: node q2.js

File: CodigoEmPython/q2.py
def contar_a(s):
    # Convertendo a string para minúscula para verificar 'a' e 'A'
    s_lower = s.lower()
    contagem = s_lower.count('a')
    return contagem

# Você pode definir a string aqui ou usar a entrada do usuário
string = input("Digite uma string: ")  # ou defina: string = "Exemplo de String com A"
contagem = contar_a(string)

print(f"A letra 'a' ocorre {contagem} vez(es) na string.")


#PARA RODAR O CODIGO NO TERMINAL: py q2.py